packages/cli/src/commands/template/build.ts

```diff
@@ -340,20 +345,23 @@ export const buildCommand = new commander.Command('build')
           true,
         )
 
-        try {
-          child_process.execSync(
-            `echo "${accessToken}" | docker login docker.${connectionConfig.domain} -u _e2b_access_token --password-stdin`,
-            {
-              stdio: 'inherit',
-              cwd: root,
-            },
-          )
-        } catch (err: any) {
-          console.error(
-            'Docker login failed. Please try to log in with `e2b auth login` and try again.',
-          )
-          process.exit(1)
+        if (opts.imageUrlMask == undefined) {
```

Comment:
why skip auth when imageUrlMask is provided?

Reply:
Because if a mask is provided,  you will never use this auth; actually, it can break stuff. Imagine that I am authorized to aws container registry, then I will run cli with the mask attribute, and here Docker will authorize to our default registry. Then docker push will fail, because iam authorized to a different registry.